app/src/editor/canvas/components/KeyboardShortcutsSidebar.jsx

```diff
@@ -161,6 +161,13 @@ class KeyboardShortcuts extends React.Component {
             return
         }
 
+        // close on esc
+        const { onClose } = this.props
```

Comment:
I guess doesn't really matter but should it be handled in `onKeyUp`?

Reply:
Had not considered keyup vs keydown here, not sure what the general convention is for that, maybe worth taking note of how esc works in other apps but AFAIK currently everything else in the streamr app acts on keydown so if we change esc handling here to happen on keyup we'd probably want to change it everywhere.

Reply:
Sure, only difference here is the key highlighting so it closes the sidebar immediately. But all good 👍